docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -35,60 +35,75 @@ Required connection parameters include the following:
 Microsoft Entra ID was previously known as Azure Active Directory (Azure AD).
 </Note>
 
-## Usage
-
-This integration can only be used to create chatbots for Microsoft Teams via the [`CREATE CHATBOT`](/agents/chatbot) syntax. Currently, *it cannot be used as a data source for other workloads*.
+### How to set up Microsoft Teams app
```

Comment:
I wonder if it is correct to put it this way. You will also be setting up the chat-bot in MindsDB during this process.

Reply:
Changed it to `How to set up Microsoft Teams app and MindsDB chatbot`